### DISCUSSION TITLE
Expanding Tutorial Resources to Cover All Ecosystem Services

### DISCUSSION THREAD

**Discussion by awellnitz-materna** [eclipse-tractusx/tutorial-resources#70] [Ideas]
Hey everyone, I've been using the tutorial resources in our repositories, and while they've been incredibly helpful for getting started, I've noticed that they only cover a few services within our ecosystem. 
As a newcomer, it would be really beneficial to have tutorials that encompass all the services so that I can get a comprehensive understanding of how everything fits together. 

Any thoughts on expanding our tutorial coverage?

**Comment by stephanbcbauer** [comment]
@awellnitz-materna thank you for this discussion. Indeed, it's already “planned” ( not planned, but we also have the idea ) to extend the MXD with e.g.

- portal
- dDTR
- ...

Contributions are welcome ;) If you have some time, feel free to add some functionality

**Reply by herm4ns** [reply]
I am wondering which components are currently included in this MXD? Does it also cover the DTR or is there a seperated tutorial?

**Reply by awellnitz-materna** [reply]
@herm4ns The following services are currently provided:
- 2 EDC (Alice & Bob)
- Keycloak
- MIW
- Hashicorp Vault
- Postgres DB
